next.config.js

```diff
@@ -1,22 +1,7 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {
-  reactStrictMode: true,
-  swcMinify: true,
-  compiler: { styledComponents: true },
-  eslint: {
-    dirs: ["src"],
-  },
-  webpack(config) {
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: ["@svgr/webpack"],
-    });
-
-    return config;
-  },
-};
+const nextConfig = {};
```

Comment:
the stock nextjs config actually has everything we need now and having superfluous configs here can actually accidentally turn off features that we do want